logger/log.go

```diff
@@ -122,7 +122,5 @@ func (l *Logger) Debugf(format string, v ...interface{}) {
 
 // Tracef logs a trace statement
 func (l *Logger) Tracef(format string, v ...interface{}) {
-	if l.trace {
```

Comment:
Instead of removing, could we add a `ForceTracef` function that is duplicate of `Tracef` but without checking `l.trace` boolean?